images/bay-php/settings.php

```diff
@@ -174,8 +174,8 @@
 // Aggregate JavaScript files on
 $config['system.performance']['js']['preprocess'] = 1;
 
-if ($smtp_on = getenv('ENABLE_SMTP')) {
-  $config['smtp.settings']['smtp_on'] = (strtolower($smtp_on) == "true");
+if (strlower(getenv('ENABLE_SMTP')) === "true") {
+  $config['system.mail']['interface']['default'] = 'SMTPMailSystem';
```

Comment:
We actually need to set Drupal's `system.mail` interface when SMTP is enabled.

https://www.drupal.org/project/smtp/issues/3003639

Reply:
Good find. This comment is particularly amusing.

> What we should do: improve documentation.